src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -131,13 +132,10 @@ void CHyprGroupBarDecoration::draw(CMonitor* pMonitor, float a, const Vector2D&
 
             refreshGradients();
 
-            if (*PGRADIENTS) {
-                CBox rect2 = rect;
-                rect2.scale(pMonitor->scale);
-                g_pHyprOpenGL->renderTexture((m_dwGroupMembers[i] == g_pCompositor->m_pLastWindow ? m_tGradientActive : m_tGradientInactive), &rect2, 1.0);
-            }
+            if (*PGRADIENTS)
+                g_pHyprOpenGL->renderTexture((m_dwGroupMembers[i] == g_pCompositor->m_pLastWindow ? m_tGradientActive : m_tGradientInactive), &rect, 1.0);
 
-            rect.y      = ASSIGNEDBOX.y + ASSIGNEDBOX.h / 2.0 - (*PTITLEFONTSIZE + 2 * BAR_TEXT_PAD) / 2.0;
+            rect.y      = (ASSIGNEDBOX.y + BAR_TEXT_PAD) * pMonitor->scale;
```

Comment:
I'm not shure about this line, because i'm not really shure what the replaced line was doing.
ASSIGNEDBOX.h / 2.0   
is just the half size of the box,
(*PTITLEFONTSIZE + 2 * BAR_TEXT_PAD) / 2.0
is half the font size + the bar text padding, isn't that just the the half size of the box again ?